CODEOWNERS

```diff
@@ -1,2 +1,6 @@
+# The ipdx team is responsible for GitHub Management maintenance (at least through 2024)
+* @ipfs/ipdx
```

Comment:
GitHub is unhappy about these changes.

Reply:
Doh - thanks - good catch!

Reply:
Fix.  Also, for my own edification, how did you learn Github was unhappy about this.  I believe CODEOWNERS had been referencing ipfs/ipdx for quite awhile and I'm wondering why it is just being spotted now.

Reply:
If you go to the PR files, you'll see a "This CODEOWNERS file contains errors" message.

Also, github still seems unhappy...